src/lib.rs

```diff
@@ -140,14 +145,17 @@ fn build_new_addr(
             administrative_regions: admins,
             weight: weight,
             zip_codes: postcodes.clone(),
-            coord: poi.coord.clone(),
-            distance: None,
+            coord: poi.coord,
+            country_codes: country_codes.clone(),
+            ..Default::default()
         },
-        label: label,
-        coord: poi.coord.clone(),
+        label: addr_label,
+        coord: poi.coord,
+        approx_coord: None,
         weight: weight,
         zip_codes: postcodes,
         distance: None,
+        country_codes: country_codes,
```

Comment:
Just `country_codes` works and is shorter. (I'm surprised `cargo fmt` didn't change it...

Reply:
done.  
`cargo clippy` does warn about it but `cargo fmt` doesn't enforce it.